### ISSUE TITLE
Refactor Sinopia email task and tests

### ISSUE THREAD

**Issue by jermnelson** [LD4P/ils-middleware#67]
No longer need to parse the `sqs-sensor` message body to retrieve email or resource uri, depends on PR #66

**Comment by jmartin-sul** [comment]
based on post-standup discussion, we think it's best to leave this till #70 is done:  the refactoring would likely leverage the `sqs-message-parse` task in the stanford DAG, since the email task pre-dates that task and does its own message parsing.  but it seems very possible that the xcom written by `sqs-message-parse` will change as part of #70, possibly upending any work that'd go into this ticket now.

**Comment by jcoyne** [comment]
@jermnelson Presumably you are talking about these lines: https://github.com/LD4P/ils-middleware/blob/main/ils_middleware/tasks/sinopia/email.py#L86-L87  Are we going to replace those data with another source? The description of this ticket doesn't make that clear.

**Comment by aaron-collier** [comment]
@jermnelson this may be a duplicate of the work in #118 / #111 that I'm working on, so I've moved it to the backlog for now. If not a duplicate can you clarify? Thanks.
